examples/DS3231_Simple/DS3231_Simple.ino

```diff
@@ -23,9 +23,15 @@ RtcDS3231<TwoWire> Rtc(Wire);
 void setup () 
 {
     Serial.begin(57600);
+    while (!Serial) delay(250);    // Wait until Arduino Serial Monitor opens
+ 
+    #if defined(ESP8266)
```

Comment:
I don't like to include "platform specific" in examples as it just makes them harder to read.